Web_Development/FrontEnd/CSS_Rounding Corners/Rounding Corners.md

```diff
@@ -0,0 +1,180 @@
+## **BORDER  RADIUS**
+
+### WHAT IS BORDER RADIUS
+
+Border Radius is an attribute that rounds up the corner of the border around an element. You can also give border radius with background image or border image as well.
+

```

Comment:
1. Not a proper way to write the content is way too little. Also, don't use words like "You", "We"..etc.